packages/tools/e2e-tests/src/fixtures/mocks/send.ts

```diff
@@ -0,0 +1,3 @@
+export const send: object = {
```

Comment:
```suggestion
export const send: Record<string, string[]> = {
```

Might not be good/specific enough, but `object` is usually too loose. Oh well, this is just a mock. And then we are forced to type stuff when declaring vars, lol. This can be turned off in ESLint settings (maybe just override for mock folders). Less work to just import the actual type if we have it?

Reply:
These objects are representations of the response objects made by the blockchain. I'm not entirely sure we can make these types much more specific.